.sync.yml

```diff
@@ -3,3 +3,4 @@ appveyor.yml:
   delete: true
 .github/workflows/ci.yml:
   beaker_facter: 'mongodb_repo_version:MongoDB:4.4,5.0,6.0,7.0'
+  unmanaged: true
```

Comment:
I tried to put multinode tests in a separate workflow, but with the same triggers it got canceled by the ci workflow ..